yomi.lua

```diff
@@ -25,7 +25,13 @@ local function yomi_config(opts)
     name = N,
     url = "",
     timeout = 5.0,
+    log_virus = true,
+    log_suspicious = true,
+    log_virus = true,
+    log_unsure = true,
     log_clean = true,
+    log_not_submitted = true,
+    log_http_return_code = true,
```

Comment:
Change default to false:
```suggestion
    log_http_return_code = false,
```